src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -456,6 +461,45 @@ protected function move(Request $request, $movement)
     }
 
     /**
+     * @param Request $request
```

Comment:
This is ok, I asked @GSadee to bring back this method. We will rework in soon, but that's tmp solution.